pyproject.toml

```diff
@@ -21,7 +21,6 @@ license = {text = "MIT"}
 classifiers = [
     "Development Status :: 3 - Alpha",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
```

Comment:
Do you want to add 3.12 to the list? I have to confess I find classifiers not that useful and and super easy to become stale and outdated.